README.md

```diff
@@ -1,65 +1,128 @@
+This program will produce data exports for submission to NCBI, UCSC, and Europe PMC.  Files provided to these resources
+are:
+* **NCBI:** Gene (NCBI gene identifier to Reactome top level pathways) and Protein (all UniProt entries in Reactome
+ associated with any NCBI Gene identifier).
+* **UCSC:** Entity (UniProt entries in Reactome) and Events (UniProt entries in Reactome in relation to Reactome 
+pathways and reactions).
+* **Europe PMC:** Profile and Link XML (describing Reactome as a provider and Reactome Pathway to literature 
+references, respectively).
+
+The file outputs will be as follows in the configured output directory (see configuration section below) and where XX 
+is the Reactome Version:
+
+* gene_reactomeXX-Y.xml (where Y is the file number as this file is split between multiple files so the file upload 
+size is acceptable for NCBI)
 * proteins_versionXX (local file that is NOT uploaded to NCBI)
 * protein_reactomeXX.ft
 * ucsc_entityXX
 * ucsc_eventsXX
 * europe_pmc_profile_reactome_XX.xml
 * europe_pmc_links_reactome_XX.xml
 
+After the files have been generated, the following files will be uploaded to external FTP Servers:
+
+* To the **NCBI FTP Server**, the "gene_reactomeXX-Y.xml" and "protein_reactomeXX.ft" files will be uploaded (using the 
+values provided in the configuration file to connect to the FTP Server - see the [Configuration](#configuration) 
+section below)
+
+* To the **EuropePMC FTP Server**, the "europe_pmc_profile_reactome_XX.xml" and "europe_pmc_links_reactome_XX.xml" 
+files will be uploaded (using the values provided in the configuration file to connect to the FTP Server - see the 
+[Configuration](#configuration) section below)
+
 ## Compiling & Running
 
-The program can be run by invoking the script `runDataExporter.sh` at the root directory of this project.  It will prompt for configuration values if none have been previously set, so the configuration documentation below is for reference or if you wish to alter existing configuration values.
+The program can be run by invoking the script `runDataExporter.sh` at the root directory of this project.
+
+The `-b or --build_jar` option can also be provided to force `data-exporter.jar` to be re-built.  The jar file
+will be built if it does not exist, regardless of specifying this option.
+
+The `-c or --overwrite_config_file` option can be provided to force resetting of the configuration values, but
+configuration values will be prompted for if a configuration file is missing or incomplete, regardless of specifying 
+this option.
 
-Optionally, using the -b or --build_jar option, the script will force `data-exporter.jar` to be re-built.  The jar file will be built regardless of specifying this option if it does not exist.
+The `-sit or --skip_integration_tests` option can be provided to skip integration tests which require connection to the
```

Comment:
SIT can often refer to "systems integration test" (https://en.wikipedia.org/wiki/System_integration_testing). Having it refer to "*skip* integrations test" will probably confuse someone later on.

Reply:
Maybe something like -skipTests, which is similar to the maven notation?

Reply:
Oh, sorry, I didn't catch that it was for skipping integration tests, and for the shorthand flag. Maybe just -s then?

Reply:
Yeah, I struggled to find a shorthand flag for skipping integration tests.  I guess I can use `-s` I was just concerned it didn't get the intent across.  

Maybe I can add a `-h|--help` flag to give info on the shell script and its options.

Reply:
Done:
https://github.com/reactome/release-data-exporter/pull/4/commits/9a21281df971a580e895081a7bdfb4aa8c245f89 (changes flag to -s)
https://github.com/reactome/release-data-exporter/pull/4/commits/ee8e6f53ce5b87ec5a9688488805abcdc96e3ea7 (adds help flag)
https://github.com/reactome/release-data-exporter/pull/4/commits/1d1215fdeaed430a779544c1696be4c23851690c
(bug fix for flipped logic in displaying help text)